trace.go

```diff
@@ -19,13 +19,15 @@ limitations under the License.
 package trace
 
 import (
+	"bytes"
 	"encoding/json"
 	"fmt"
 	"path/filepath"
 	"runtime"
 	"strings"
 	"sync/atomic"
 
+	"github.com/alecthomas/template"
```

Comment:
the less deps in trace, the better, do we really needs templating vs just text/template